### PR TITLE
Add ingress_hostname to openshift

### PR DESCRIPTION
**Pull request by jdorel**
##### SUMMARY

Add `ingress_hostname` to Openshift for route configuration.

~~Also renames `kubernetes_ingress_hostname` to `ingress_hostname` for easier configuration.~~

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - Installer

##### ADDITIONAL INFORMATION

Maybe the change of `kubernetes_ingress_hostname` to `ingress_hostname` isn't worth the breaking change of variable name ?

### CODE REVIEW

File: installer/inventory [hunk 1]
```diff
@@ -28,10 +28,11 @@ dockerhub_base=ansible
 # pg_cpu_limit=1000
 # pg_mem_limit=2
 
+# Ingress Configuratio
```
Comment (shanemcd) [inline installer/inventory]: Missing an `n` here?

File: installer/inventory [hunk 2]
```diff
@@ -28,10 +28,11 @@ dockerhub_base=ansible
...
+ingress_hostname=awx.example.org
```
Comment (shanemcd) [inline installer/inventory]: Other default variables in this file are commented out. We should probably do that here too.

**Review by shanemcd — CHANGES_REQUESTED** [review]
I think I would like to keep the variable name the same, to prevent breaking current installs.

**Review by ryanpetrello — APPROVED** [review]
any more thoughts on this one @shanemcd ?

**Comment by ryanpetrello** [comment]
It looks like this was effectively added via: 52712a0d9a4

**Pull request closed without merging**